Client/src/Remote/Endpoints/GraphStateEndpoint.tsx

```diff
@@ -23,4 +23,9 @@ export const callCreateGraphState = async (graphState: IGraphStateModel): Promis
 export const callUpdateGraphState = async (graphState: IGraphStateModel): Promise<string> => {
   const result = await put(graphStateRoute).withBody(graphState).json()
   return result
+}
+
+export const callDeleteGraphState = async (graphState: IGraphStateModel, userID: number) => {
+  const result = await _delete(graphStateRoute + '/' + graphState.id).withBody({ userID }).json()
+  return result
 }
```

Comment:
I am not sure if this is a problem, but I think you need to have an empty line at the end of the file

Reply:
why is the id being passed in the body? i thought that the backend can get it from the jwt that is passed, so that you dont have to pass the id yourself

Reply:
yeah I checked the code you shouldnt be providing your userId here so you can safely remove it everywhere else, check it should still work